src/widgets/tabs.rs

```diff
@@ -131,6 +140,70 @@ impl<'a> Tabs<'a> {
         self.divider = divider.into();
         self
     }
+
+    /// Sets the padding between tabs.
+    ///
+    /// # Examples
+    ///
+    /// A space on either side of the tabs.
+    /// ```
+    /// # use ratatui::{prelude::*, widgets::Tabs, symbols};
+    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding(" ", " ");
+    /// ```
+    /// Nothing on either side of the tabs.
+    /// ```
+    /// # use ratatui::{prelude::*, widgets::Tabs, symbols};
+    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).padding("", "");
+    /// ```
+    pub fn padding<T, U>(mut self, left: T, right: U) -> Tabs<'a>
+    where
+        T: Into<Line<'a>>,
+        U: Into<Line<'a>>,
+    {
+        self.padding_left = left.into();
+        self.padding_right = right.into();
+        self
+    }
+
+    /// Sets the left side padding between tabs.
+    ///
+    /// # Example
+    ///
+    /// An arrow on the left of tabs.
+    /// ```
+    /// # use ratatui::{prelude::*, widgets::Tabs, symbols};
+    /// let tabs = Tabs::new(vec!["Tab 1", "Tab 2"]).left_padding("->");
+    /// ```
+    pub fn left_padding<T>(mut self, padding: T) -> Tabs<'a>
```

Comment:
nit: I think I prefer left/right at the end. What do you think?
```suggestion
    pub fn padding_left<T>(mut self, padding: T) -> Tabs<'a>
```